src/components/hero/Hero.tsx

```diff
@@ -0,0 +1,28 @@
+import { Koros, Button } from "hds-react";
+import Link from "next/link";
+
+import { LandingPage } from "../../types";
+import styles from "./hero.module.scss";
+
+function Hero({ title, link, desktopImage }: LandingPage) {
+  return (
+    <div
+      className={styles.hero}
+      style={{ backgroundImage: `url(${desktopImage.uri})` }}
+    >
+      <div className={styles.box}>
+        <p>Kokosimme yhteen</p>
+        <h1>{title}</h1>
```

Comment:
Consider using https://github.com/City-of-Helsinki/liikunta-helsinki/tree/main/src/components/text for text components. That way they are consistently styled across pages.

Reply:
Will do, it skipped my mind this was already added.